auth/lib/models/code.dart

```diff
@@ -205,7 +205,7 @@ class Code {
     try {
       return int.parse(uri.queryParameters['digits']!);
     } catch (e) {
-      if (issuer.toLowerCase() == "steam") {
+      if (issuer.toLowerCase() == "steam" || issuer.toLowerCase().contains('steampowered.com')) {
```

Comment:
Can we also check host here like _getType instead of checking individual issuer? Also remove the issuer parameter if not used elsewhere in this function.

Reply:
Done! Please check the new commit. Updated _getDigits to use uri.host instead of issuer and removed the issuer parameter from the function. This should be checked though. Unfortunately, I don't have time to go deeper into the code.